docs/guides/Redux-Integration.md

```diff
@@ -2,16 +2,16 @@
 
 To handle your app's navigation state in redux, you can pass your own `navigation` prop to a navigator. Your navigation prop must provide the current state, as well as access to a dispatcher to handle navigation options.
 
-With redux, your app's state is defined by a reducer. Each navigation router effectively has a reducer, called `getStateForAction`. The following is a minimal example of how you might use navigators within a redux application:
+With redux, your app's state is defined by a reducer. Each navigation router effectively has a reducer, called `getStateForAction`. You can dispatch `NavigationActions.init` to initialize your router state.
+
+The following is a minimal example of how you might use navigators within a redux application:
 
 ```
-import { addNavigationHelpers } from 'react-navigation';
+import { addNavigationHelpers, NavigationActions } from 'react-navigation';
 
 const AppNavigator = StackNavigator(AppRouteConfigs);
 
-const initialState = AppNavigator.router.getStateForAction(AppNavigator.router.getActionForPathAndParams('Login'));
```

Comment:
I commented on your original issue. Could we actually just change this to `const initialState = AppNavigator.router.getStateForAction(NavigationActions.navigate({ routeName: 'Login' }));`? This one line may be the only change necessary (I hope!)

https://github.com/react-community/react-navigation/issues/937#issuecomment-297166585

Reply:
I had an issue with nested navigator and the docs' proposed solution for Redux integration, and I fixed it by simply replacing what you proposed @cooperka. It would be great to integrate just that to begin with. ;)